idunn/api/poi.py

```diff
@@ -8,7 +8,7 @@
 ]
 
 def build_blocks(es_poi, lang):
-    return [c.from_es(es_poi, lang) for c in BLOCKS_ORDER]
+    return [c.from_es(es_poi, lang) for c in BLOCKS_ORDER if c.from_es(es_poi, lang)]
```

Comment:
this will create twice the poi, I think it's better to filter the none afterward (or use a normal loop :stuck_out_tongue_winking_eye: )